Source/player.h

```diff
@@ -330,7 +333,16 @@ void InitPlayerGFX(int pnum);
 void InitPlrGFXMem(int pnum);
 void FreePlayerGFX(int pnum);
 void NewPlrAnim(int pnum, BYTE *Peq, int numFrames, int Delay, int width);
+void NewPlrAnim(int pnum, BYTE *Peq, int numFrames, int Delay, int width, int numSkippedFrames);
+void NewPlrAnim(int pnum, BYTE *Peq, int numFrames, int Delay, int width, int numSkippedFrames, int stopDistributingAfterFrame);

```

Comment:
Declare a single function with 2 default arguments?

Reply:
i'm okay with both. I changed it. thx :)